front-end/package.json

```diff
@@ -1,46 +1,49 @@
 {
-  "name": "front-end",
+  "name": "my-app",
```

Comment:
Why did we use my-app instead of front-end or S.C.I.L.E.R.?

Reply:
we could do `sciler`, however, S.C.I.L.E.R. will not work easily in testing

Reply:
This should indeed be front-end or S.C.I.L.E.R.